### ISSUE TITLE
unknown types, false positive, LIKE REF TO

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1706]
```abap
TYPES: BEGIN OF ty_struc,
         name TYPE string,
       END OF ty_struc.
DATA tab TYPE STANDARD TABLE OF ty_struc WITH DEFAULT KEY.
DATA row LIKE LINE OF tab.
DATA ref LIKE REF TO row.
row-name = 'bar'.
APPEND row TO tab.
READ TABLE tab REFERENCE INTO ref WITH KEY name = 'bar'.
WRITE ref->name.
```

![image](https://user-images.githubusercontent.com/5888506/105689257-d0a15100-5efa-11eb-86da-2999963a6795.png)
